src/chapters/orm-relationships/many-to-one.rst

```diff
@@ -0,0 +1,205 @@
+Creating a Many-to-One Relationship
+===================================
+
+The first relationship we implement will be between the ``Event`` and ``EventCategory`` classes. We will allow multiple events to be in the same category, but each event will only have one category. Thus, this will be a many-to-one relationship.
+
+Before diving in, let's reduce repetition in our persistent classes, that is, let's "DRY out" our code.
```

Comment:
i'd stay away from the quotations around DRY out - our style guide warns against using scare quotes in this fashion